assets/js/modules/analytics-4/datastore/api.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * `modules/analytics-4` data store: api.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+
+const baseInitialState = { };
+const baseActions = {};
+const baseControls = {};
+const baseReducer = ( state, { type } ) => {
+	switch ( type ) {
+		default: {
+			return state;
+		}
+	}
+};
+const baseResolvers = {};
+const baseSelectors = {
+	/**
+	 * Checks if the Admin API is working.
+	 *
+	 * Returns false if there are any errors for getProperties or getWebDataStreams.
+	 *
+	 * Returns true if getProperties or getWebDataStreams has fetched successfully.
+	 *
+	 * Else returns undefined.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {(boolean | undefined)} True, false, or undefined.
+	 */
+	isAdminAPIWorking( state ) {
+		for ( const errorKey in state.errors ) {
+			if (
+				errorKey.startsWith( 'getProperties' ) ||
+				errorKey.startsWith( 'getWebDataStreams' )
```

Comment:
The IB mentions that the keys should start with `getProperties::` not `getProperties`, so I think this is more appropriate:

```suggestion
				errorKey.startsWith( 'getProperties::' ) ||
				errorKey.startsWith( 'getWebDataStreams::' )
```